src/components/Search/Form.js

```diff
@@ -0,0 +1,63 @@
+import { render } from '../../shared/dom';
+import { redirect } from '../../';
+import { routes } from '../../route';
+import ConsoleLogger from '../../utils/ConsoleLogger';
+
+const IDLE_TIMEOUT_LIMIT_MS = 800;
+const IDLE_TIMEOUT_CHECK_MS = 50;
+
+export const SearchForm = ({ renderOn, searchInputValue }) => {
+    const template = `
+        <div class="search-form" id="search-form">
+            <input
+                type="text"
+                id="search-keywords"
+                placeholder="WYSZUKAJ"
```

Comment:
Also that should be in english, and it's not visible at all when i unfocus the input and leave it empty

Reply:
i don;t know how to fix the placeholder being invisible. could you give me some hint? :P

Reply:
The property is here, it's present, but I think that the default styling color is almost the same as input bg color, just change the placeholder color in css :P (it was input::placeholder{color: ....} i think)